apps/trading/e2e/tests/market_lifecycle/test_market_lifecycle.py

```diff
@@ -10,6 +11,8 @@
 
 @pytest.mark.usefixtures("vega", "page", "proposed_market", "risk_accepted")
 def test_market_lifecycle(proposed_market, vega: VegaService, page: Page):
```

Comment:
The characteristics of the checks in this tests are something I would expect to see in a test of a particular module, not e2e. Am I missing something? How comes this has to be e2e?

Normally I would expect to mock a state for the given ui module, and then check the default values as we do here. Is this not something we do atm?

Reply:
To get these tests into jest would be quite difficult to mock as there are multiple different data sources. It would be possible but it's a lot easier to do it here

Reply:
sure